torch/csrc/cuda/Module.cpp

```diff
@@ -173,6 +173,34 @@ PyObject * THCPModule_cudaHostAllocator(PyObject *_unused, PyObject *noargs)
   END_HANDLE_TH_ERRORS
 }
 
+PyObject * THCPModule_cudaCachingAllocator_raw_alloc(PyObject *_unused, PyObject *args){
+  HANDLE_TH_ERRORS
+  PyObject* size_o = nullptr;
+  PyObject* stream_o = nullptr;
+  if(!PyArg_ParseTuple(args, "OO", &size_o, &stream_o)) {
+    THPUtils_invalidArguments(
+        args,
+        nullptr,
+        "caching_allocator_alloc",
+        1,
+        "(ssize_t size, intptr_t stream);");
+    return nullptr;
+  }
+  ssize_t size = PyLong_AsSsize_t(size_o);
+  cudaStream_t stream = static_cast<cudaStream_t>(PyLong_AsVoidPtr(stream_o));
+  void* mem = c10::cuda::CUDACachingAllocator::raw_alloc(size);
```

Comment:
you are still calling raw_alloc and not raw_external_alloc here?

Reply:
Good catch, I swear I was not drunk.